mycroft/skills/media/vocab/__init__.py

```diff
@@ -0,0 +1,130 @@
+from os.path import join, dirname
+
+from adapt.intent import IntentBuilder
+from mycroft.skills import time_rules
+from mycroft.skills.core import MycroftSkill
+from mycroft.messagebus.message import Message
+from mycroft.configuration.config import ConfigurationManager
+
+from mycroft.util.log import getLogger
+import mopidy
+
+logger = getLogger(__name__)
+
+__author__ = 'forslund'
+
+
+class MediaSkill(MycroftSkill):
+    """
+        The MediaSkill class is a base class for media skills containing
+        vocabulary and intents for the common functions expected by a media
+        skill. In addition event handlers to lower volume when mycroft starts
+        to speak and raise it again when (s)he stops.
+
+        But wait there is one more thing! A small event emitter and matching
+        handler to stop media currently playing when new media is started.
+    """
+    def __init__(self, name):
+        super(MediaSkill, self).__init__(name)
+        self.isPlaying = False
+        config = ConfigurationManager.get_config()
+        self.base_conf = config.get('Media')
+
+    def initialize(self):
+        logger.info('Initializing MediaSkill commons')
+        logger.info('loading vocab files from ' + join(dirname(__file__),
+                    'vocab', self.lang))
+        self.load_vocab_files(join(dirname(__file__), 'vocab', self.lang))
+
```

Comment:
Is this intended to load common vocab files for all media skills that inherit from this base class, or just the vocab from the current media skill that's inheriting? If it's the latter, it might be better to just leave this to the inherited class.